lib/core/api.js

```diff
@@ -279,11 +293,17 @@ api.upload.toPlatform = function(data, sessionInfo, progress, groupId, cb) {
   var post_and_progress = function (data, callback) {
     progress(nblocks++ * 100.0 / blocks.length);
     //off to the platfrom we go
-    return postBlockToPlatform(data, groupId, nblocks, callback);
+    return postBlockToPlatform(data, groupId, nblocks, false, callback);
+  };
+
+  var post_and_progress_dataservices = function (data, callback) {
+    progress(nblocks++ * 100.0 / blocks.length);
+    //off to the dataservices platfrom we go
+    return postBlockToPlatform(data, groupId, nblocks, true, callback);
   };
 
   var post_upload_meta = function (uploadMeta, callback) {
-    return postBlockToPlatform(uploadMeta, groupId, 0, callback);
+    return postBlockToPlatform(uploadMeta, groupId, 0, false, callback);
```

Comment:
And here this raises an additional question for @jh-bate and @darinkrauss. How are we going to enforce the minimum uploader version with the new "dataservices" API? Do we want to post a second upload metadata? I actually think this would be a good idea as it also relates to the problem I was posing in Slack earlier: how will we be able to trace data (for auditing purposes) back to either jellyfish or the new dataservices? If we do a separate upload metadata POST for each (and include information about where we're posting *to* in the metadata, in a new field of some sort), then we'll have that link easily via the `uploadId` (as long as we don't allow a situation to arise again where we end up not storing upload metadata even when the rest of the POSTs go through 😉).